EvoS.Framework/DataAccess/Daos/TrustWarDao.cs

```diff
@@ -0,0 +1,33 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using Newtonsoft.Json;
+
+namespace EvoS.Framework.DataAccess.Daos
+{
+    public interface TrustWarDao
+    {
+        public TrustWarDaoEntry? Find();
+        public void Save(TrustWarDaoEntry entry);
+
+        public class TrustWarDaoEntry
+        {
+            [BsonId] public ObjectId _id = ObjectId.GenerateNewId();
+            public long Omni;
+            public long Evos;
+            public long Warbotics;
```

Comment:
Doesn't it make more sense as an array? You have to map them to their indices before using them.